django/core/files/utils.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, file):
     readline = property(lambda self: self.file.readline)
     readlines = property(lambda self: self.file.readlines)
     seek = property(lambda self: self.file.seek)
+    seekable = property(lambda self: self.file.seekable)
```

Comment:
It actually *does* get added, it's just that `lamdba self: self.file.seekable` raises an AttributeError and therefore `hasattr(file, 'seekable')` returns false in py2.